allensdk/test/brain_observatory/vbn_2022/metadata_writer/test_lims_queries.py

```diff
@@ -0,0 +1,65 @@
+import pandas as pd
```

Comment:
This test file needs a unique name. I created another one called `test_vbn_2022_metadata_writer_lims_queries.py`

Reply:
Didn't see that. Thanks. I've moved the test over into that file.